core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -36,9 +37,7 @@
   private static final List<BasicUnit> scalar = Arrays.asList(BasicUnit.SCALAR);
   private static final List<BasicUnit> seconds = Arrays.asList(BasicUnit.SECONDS);
 
-  /**
-   * Census defined rpc client {@link MeasurementDescriptor}s.
-   */
+  // Census defined rpc client {@link MeasurementDescriptor}s.
```

Comment:
As Kristen notes, the javadoc only associates with one definition. Eventually we should probably document each of the views but that should probably happen in the proto-based version of this file once we have finalized all of the view.